components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/util/AuthenticatorMgtExceptionBuilder.java

```diff
@@ -121,7 +121,9 @@ public enum AuthenticatorMgtError {
         ERROR_CODE_DELETING_ENDPOINT_CONFIG("65012", "Error while managing endpoint configurations.",
                 "Error while managing endpoint configurations for the user defined local authenticator %s."),
         ERROR_CODE_HAVING_MULTIPLE_PROP("65013", "Multiple properties found", "Only actionId " +
-                "property is allowed for authenticator: %s.");
+                "property is allowed for authenticator: %s."),
+        ERROR_WHILE_CHECKING_FOR_EXISTING_AUTHENTICATOR_BY_NAME("65014", "Error while retrieving " +
+                "authenticator.", "Error while check any authenticator exists by given name: %s.");
```

Comment:
Error while check any authenticator exists by given name -> Error while checking if an authenticator exists with the given name